### PR TITLE
DFU Firmware Update

### PR DESCRIPTION
**Pull request by mobilegurudevelopment**

### CODE REVIEW

File: station.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -7574,7 +8166,7 @@
 				);
 				MARKETING_VERSION = 0.7;
 				OTHER_SWIFT_FLAGS = "$(inherited) -D COCOAPODS";
-				PRODUCT_BUNDLE_IDENTIFIER = com.ruuvi.station;
+				PRODUCT_BUNDLE_IDENTIFIER = sg.megapixel.test;
```
Comment (rinat-enikeev) [inline station.xcodeproj/project.pbxproj:8169]: this should not go into main repo

**Review by rinat-enikeev — CHANGES_REQUESTED** [review]

**Comment by rinat-enikeev** [comment]
IMG_52AC70162DB3-1

Missing localisation for "Open document picker"

**Comment by rinat-enikeev** [comment]
IMG_393D8B1B364F-1
missing i18n for buttons

**Comment by rinat-enikeev** [comment]
IMG_4DA1BE75DA43-1
missing i18n for "Finish"

**Comment by rinat-enikeev** [comment]
User should not be able to swipe down the view controller during the DFU process. 

Please check how it is done for Discover module here: 
https://github.com/ruuvi/com.ruuvi.station.ios/blob/8d48c588656065aed818bf92a3a527b39eacc7c9/station/Classes/Presentation/Modules/Dashboard/Cards/Router/CardsRouter.swift#L41-L41

https://github.com/ruuvi/com.ruuvi.station.ios/blob/8d48c588656065aed818bf92a3a527b39eacc7c9/station/Classes/Presentation/Modules/Dashboard/Cards/Router/CardsRouter.swift#L110

**Comment by mobilegurudevelopment** [comment]
> User should not be able to swipe down the view controller during the DFU process.
> 
> Please check how it is done for Discover module here:
> https://github.com/ruuvi/com.ruuvi.station.ios/blob/8d48c588656065aed818bf92a3a527b39eacc7c9/station/Classes/Presentation/Modules/Dashboard/Cards/Router/CardsRouter.swift#L41-L41
> 
> https://github.com/ruuvi/com.ruuvi.station.ios/blob/8d48c588656065aed818bf92a3a527b39eacc7c9/station/Classes/Presentation/Modules/Dashboard/Cards/Router/CardsRouter.swift#L110

Updated

File: station.xcodeproj/project.pbxproj (hunk 1 above)
Comment (mobilegurudevelopment) [reply station.xcodeproj/project.pbxproj]: solved

**Comment by mobilegurudevelopment** [comment]
@rinat-enikeev I solved your mentioned issues, please help to recheck. Thanks

**Comment by rinat-enikeev** [comment]
`Next` button on First screen is not translated, `Progress` text on Flash screen is not translated.

**Comment by mobilegurudevelopment** [comment]
> `Next` button on First screen is not translated, `Progress` text on Flash screen is not translated.

updated

**Pull request merged**